File: src/hello_world/hello_world.erl
-module(hello_world).

% elp:ignore W0012 W0040 E1599
-moduledoc {file, "../../docs/hello_world.md"}.

-export([
    start/0
]).

start() ->
    temporal_sdk:start_workflow(cluster_1, "default", hello_world_workflow, [
        wait, {input, ["from Temporal"]}
    ]).

File: lib/hello_world/hello_world.ex
defmodule HelloWorld do
  @external_resource "docs/hello_world.md"
  @moduledoc TemporalSdk.Utils.Code.exdoc!("docs/hello_world.md")

  def start do
    TemporalSdk.start_workflow(:cluster_1, "default", HelloWorld.Workflow, [
      :wait,
      input: [["from Temporal"]]
    ])
  end
end
